analyzer/uncategorized/evm_raw_tx.go

```diff
@@ -25,7 +25,7 @@ func decodeEthRawTx(body []byte) (*sdkTypes.Transaction, error) {
 	}
 	chainIDBI := ethTx.ChainId()
 	signer := ethTypes.LatestSignerForChainID(chainIDBI)
-	pubUncompressed, err := LondonSenderPub(signer, &ethTx)
+	pubUncompressed, err := CancunSenderPub(signer, &ethTx)
```

Comment:
This is unrelated to the rest of the PR right? Just future-proofing against a time when our EVM-compatible runtimes start using cancun-style txs?

Reply:
oh yeah, it's separate. I ended up revisiting the somewhat-vendored code when it looked like I would have to vendor some more for un-weird event topic parsing